pose_format/pose_body.py

```diff
@@ -12,7 +12,7 @@
 class PoseBody:
     tensor_reader = 'ABSTRACT-DO-NOT-USE'
 
-    def __init__(self, fps: float, data, confidence):
+    def __init__(self, fps: int, data, confidence):
```

Comment:
`fps` is indeed a `float` (as it can be for videos, and in fact usual 30fps videos are 29.97 for example)